Project.toml

```diff
@@ -1,9 +1,10 @@
 name = "DrWatson"
 uuid = "634d3b9d-ee7a-5ddf-bec9-22491ea816e1"
 repo = "https://github.com/JuliaDynamics/DrWatson.jl.git"
-version = "2.4.0"
+version = "2.4.1"
 
 [deps]
+DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
```

Comment:
This must be in `[extras]`, not in `[deps]`

Reply:
Correct, I was having issues getting it there, but apparently I just needed to add it to the target list. This should all actually be fixed now